clang/test/CodeGen/ubsan-src-ignorelist-category.test

```diff
@@ -0,0 +1,37 @@
+// RUN: rm -rf %t
```

Comment:
Consider pre-committing this test (with the "bad" output) as a separate patch, so that the current patch will show how the test output is fixed.

Reply:
Sounds Good. 
https://github.com/llvm/llvm-project/pull/140602

Reply:
THe point of SPR to  have stacked PR, so this one will already be rebased on #140602

Reply:
But in this case we failed to do so, if reordering of patches is needed, SPR often fails.